sdk/src/manifest.rs

```diff
@@ -1234,6 +1234,29 @@ impl Manifest {
             .await
     }
 
+    /// Embed a signed manifest into fragmented BMFF content (i.e.DASH) assets using a supplied signer.
+    #[cfg(feature = "file_io")]
+    pub fn embed_to_bmff_fragmented<P: AsRef<Path>>(
```

Comment:
We should probably just add this to Builder instead of Manifest at this point, but whatever works for a demo. Does the sequence of fragments matter? 
We could support this via streaming with an archive, or another thing I've been working on, - a ResourceResolver. In that case you'd pass in a list of identifiers, and a resolver, which acts like a trait for ResourceStore.